nomos-mix/message/src/lib.rs

```diff
@@ -4,7 +4,7 @@ pub use error::Error;
 
 use sha2::{Digest, Sha256};
 
-pub const MSG_SIZE: usize = 1024;
+pub const MSG_SIZE: usize = 2048;
```

Comment:
why are we doubling this?

Reply:
It's because the size of block proposal is between 1024 and 2048 currently in our tests. But, this file is just a mock, and we haven't determined the proper block size yet. So, I'm using just 2028 here for now.

Reply:
I think it's better to put this in the configuration.